### PR TITLE
fix(core): bump swc to 5.0.1

### PR DESCRIPTION
**Pull request by hardfist**
## Summary
bump swc to 5.0.1 to fix https://github.com/swc-project/swc/pull/9720

## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -79,14 +80,16 @@ inventory = { version = "0.1" }
 rkyv      = { version = "=0.8.8" }
 
 # Must be pinned with the same swc versions
+swc                 = { version = "5.0.0", features = ["manual-tokio-runtmie"] }
```
Comment (chenjiahan) [inline Cargo.toml:83]: runtime typo?

**Comment by hardfist** [comment]
!eco-ci

File: Cargo.toml (hunk 1 above)
Comment (hardfist) [reply Cargo.toml:83]: it'a typo in swc, so we need to follow this typo

**Comment by hardfist** [comment]
!eco-ci

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**